sql/mysqld.cc

```diff
@@ -1729,28 +1729,14 @@ static void close_connections(void)
 
   /* Abort listening to new connections */
   DBUG_PRINT("quit",("Closing sockets"));
-  if (!opt_disable_networking )
+  for (std::vector<MYSQL_SOCKET>::iterator sock = listen_sockets.begin();
+       sock != listen_sockets.end(); ++sock)
```

Comment:
I presume it was tested in buildbot and passed on all builders, so whenever I start thinking "does it compile on all our ancient compilers" I'll assume it was tested and it does.

Reply:
I don't think I understand your words here. Could you please clarify?

STL containers and iterators are from C++98, so I guess it's supported by any C++ compiler alive? Or do you suggest changing this to a range-based for loops? Those require C++11. And probability of having a compiler which doesn't support C++11 is significantly higher.